public/assets/styles/_sections/Support.less

```diff
@@ -0,0 +1,80 @@
+#Support {
+  .mainImage {
+    background-image: url('../images/Support_image_min.jpg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    width: 120%;
+    margin-left: -11%;
+    img {
+      opacity: 0;
```

Comment:
Just a reminder about what we talked about...

However clever, let's not use this technique of adding both an `<img>` tag and a css `background-image`, as that will add an extraneous network request. Instead, just use an `<img>` tag, or just use `background-image` and add a height a width, and use `background-size` to crop image correctly.